client/src/pages/Genres.js

```diff
@@ -3,20 +3,23 @@ import styled from '@emotion/styled';
 import Button from '../components/Button';
 import Content from '../components/layout/Content';
 import Form from '../components/layout/Form';
+import GetGenres from '../components/ChooseGenres';
 
 const Text = styled.div`
   display: flex;
   flex-wrap: wrap;
   text-align: center;
   font-family: 'Quicksand Light';
   margin-top: 80px;
+  margin-bottom: 25px;
 `;
 
 function Genres() {
   return (
     <Content>
       <Form>
         <Text>Which Music Do You Like?</Text>
+        <GetGenres />
         <Button size="Large">Match Me</Button>
```

Comment:
I know it's not part of this PR, but this button has a `height` declaration that you don't need, which makes the text be off-center vertically. :)

Reply:
I'll put that on my todo list